keps/sig-storage/1790-recover-resize-failure/README.md

```diff
@@ -107,28 +108,68 @@ As part of this proposal, we are mainly proposing three changes:
    - NodeExpansionFailed // state set when expansion has failed in kubelet with a terminal error. Transient errors don't set NodeExpansionFailed.
 3. Update quota code to use `max(pvc.Spec.Resources, pvc.Status.AllocatedResources)` when evaluating usage for PVC.
 
+### Making resizeStatus more general
```

Comment:
Add a note here that this is a change being proposed for 1.27. Also add it to the history section.

I still found a few references to `ResizeStatus`.

Reply:
Added and removed the references.